docs/sql-data-sources-avro.md

```diff
@@ -177,6 +180,19 @@ Data source options of Avro can be set using the `.option` method on `DataFrameR
   Currently supported codecs are <code>uncompressed</code>, <code>snappy</code>, <code>deflate</code>, <code>bzip2</code> and <code>xz</code>.<br> If the option is not set, the configuration <code>spark.sql.avro.compression.codec</code> config is taken into account.</td>
     <td>write</td>
   </tr>
+  <tr>
+    <td><code>mode</code></td>
+    <td>FAILFAST</td>
+    <td>The <code>mode</code> option allows to specify parse mode for function <code>from_avro</code>.<br>
+      Currently supported modes are:
+      <ul>
+        <li><code>FAILFAST</code>: Throws an exception on processing corrupted record.</li>
+        <li><code>PERMISSIVE</code>: Corrupt records are processed as null result. To implement this, the
```

Comment:
not a big deal but ..`To implement this` sounds a little bit awkward .. I think you can just say ... `Corrupt records are processed as null; therefore, the given schema is forced to be fully nullable.` (feel free to fix words as you think righter).

Reply:
I see. Let me update it.